appeals/web/src/server/appeals/appeal-details/address/__tests__/address.test.js

```diff
@@ -0,0 +1,215 @@
+import { parseHtml } from '@pins/platform';
+import supertest from 'supertest';
+import { appealData } from '#testing/app/fixtures/referencedata.js';
+import { createTestEnvironment } from '#testing/index.js';
+import nock from 'nock';
+
+const { app, installMockApi, teardown } = createTestEnvironment();
+const request = supertest(app);
+const baseUrl = '/appeals-service/appeal-details';
+
+describe('neighbouring-sites', () => {
+	beforeEach(installMockApi);
+	afterEach(teardown);
+
+	describe('GET /add', () => {
+		it('should render updateSiteAddress page', async () => {
+			const appealId = appealData.appealId.toString();
+			const response = await request.get(
+				`${baseUrl}/${appealId}/appellant-case/site-address/change/1`
+			);
+			const element = parseHtml(response.text);
+
+			expect(element.innerHTML).toMatchSnapshot();
```

Comment:
it may be worth adding a quick `toContain` check for the `h1` content or something else to identify the page, just to guard against the possibility of the snapshot changing to an error page or something in future

Reply:
Done